test/Python/python_runtime.swift

```diff
@@ -6,17 +6,43 @@
 import Python
 import StdlibUnittest
 
+let gc = Python.import("gc")
+// Python3 exclusive:
```

Comment:
Doc comment:
```suggestion
/// The tracemalloc module is a debug tool to trace memory blocks allocated by
/// Python. It is optionally imported because it is available only in Python 3.
```